activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/converter/ProcessModelContentConverter.java

```diff
@@ -78,7 +79,11 @@ public Optional<BpmnProcessModelContent> convertToModelContent(BpmnModel bpmnMod
 
     public BpmnModel convertToBpmnModel(byte[] modelContent) {
         try (InputStreamReader reader = new InputStreamReader(new ByteArrayInputStream(modelContent))) {
-            XMLStreamReader xmlReader = createSafeXmlInputFactory().createXMLStreamReader(reader);
+            XMLInputFactory safeXmlInputFactory = createSafeXmlInputFactory();
```

Comment:
createSafeXmlInputFactory() method is defined in other module, not sure if CodeQL is able to check code there. Also they recommend other solution, but result should be the same.